cke-tools/src/README.md

```diff
@@ -18,3 +19,27 @@ CKE tools is licensed under MIT license.
 
 [godoc]: https://godoc.org/github.com/cybozu/neco-containers/cke-tools/src
 [CKE]: https://github.com/cybozu-go/cke
+
+`updateblock117`
+----------------
```

Comment:
Move this section before the list of markdown references.
References should always placed at the end of the document.